lib/srv/db/proxyserver.go

```diff
@@ -342,6 +343,15 @@ func (s *ProxyServer) handleConnection(conn net.Conn) error {
 	return nil
 }
 
+// getMySQLVersionFromServer returns the MySQL version returned by an instance on last connection or
+// the MySQLServerVersion set in configuration if the first one is not available.
+// Function picks a random server each time if more than one are available.
+func getMySQLVersionFromServer(servers []types.DatabaseServer) string {
+	count := len(servers)
+	db := servers[rand.Intn(count)].GetDatabase()
+	return db.GetMySQLServerVersion()
```

Comment:
Does it is possible that two different agents servers contain different `mysql version`  ?

Reply:
Or it will be great if we can propagate the version change to all servers, if possible.

Reply:
Added, kind of https://github.com/gravitational/teleport/pull/12196#issuecomment-1111770126